### PR TITLE
Added transitions with Animate.css

### PR DESCRIPTION
**Pull request by presidenten**
Added nice forwards and backwards transitions that can use any of the animations in Animate.css.

Examples here (try navigating forward and back):
http://presidenten.github.io/remark/slideleft.html
http://presidenten.github.io/remark/slideup.html
http://presidenten.github.io/remark/rotateOut.html

I did not now how to best add the styling that is needed though, and I suspect that the best place is not in the template in README.md. Feel free to move it.

Hope you like my addition to remark.js!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +43,41 @@ Below is a boilerplate HTML file to get you started:
...
+       */  
+       
+       /* Browsing forwards transition */
+    
+       /* Browse to slide */
+       .remark-visible.remark-browsing-backwards {
+           animation: slideInDown 0.5s forwards;
+       }
+       /* Browse from slide */
+         .remark-fading.remark-browsing-backwards {
```
Comment (darabos) [inline README.md]: The indentation is a bit off on some lines. It's a fantastic change otherwise! I hope it gets merged.

**Comment by darabos** [comment]
I just discovered that animated transitions are already supported! Check out https://github.com/gnab/remark/pull/69. You just include `animate.css` and add `class: animated, slideInRight` to a layout slide, and everything is animated then!

I think the main problem is that this is not documented anywhere as far as I can see.

Your change seems to work better though when toggling between two slides quickly. And it supports opposite animations for stepping left and right. Well, good luck with it!

**Comment by oz123** [comment]
Maybe you should just add a subsection to the wiki, https://github.com/gnab/remark/wiki/Markdown#slide-properties

**Comment by utdrmac** [comment]
What's the status of this?

**Comment by utdrmac** [comment]
@presidenten Since this is already part of remark, can you cancel/close this PR please?

**Pull request closed without merging**